lib/logger/src/helpers.ts

```diff
@@ -115,9 +115,8 @@ export function waitOnExit(process: string, child: ChildProcess): Promise<void>
       if (code === 0) {
         resolve()
       } else {
-        const error = new ToolKitError(`${process} process returned an error`)
-        child.stderr?.setEncoding('utf8')
-        error.details = child.stderr?.read()
+        const error = new ToolKitError(`${process} process returned a non-zero exit code (${code})`)
+        error.details = 'error output has been logged above ☝️'
```

Comment:
**question**: would it make sense to assign the value to error.details if read() returns a non null/undefined value
ie. 
```error.details = child.stderr?.read() ?? 'error output has been logged above ☝️'```

Reply:
it's never going be defined as we should always be catching all the output and printing it via winston